.travis.yml

```diff
@@ -292,17 +292,31 @@ matrix:
 
     - os: linux
       dist: xenial
-      name: GCC 5 (ARM64)
+      name: GCC 5 (ARM64), C++11
```

Comment:
We are not using travis anymore, we should probably remove this file. Please add the workflow to the github actions instead (`.github/workflows/LinuxRelease.yml`).